crates/api_crud/src/user/create.rs

```diff
@@ -6,22 +6,14 @@ use lemmy_api_common::{
   utils::{blocking, honeypot_check, password_length_check, send_verification_email},
 };
 use lemmy_apub::{
-  generate_followers_url,
```

Comment:
I think the commit hook would reject this (did it run for  you when you committed this?) - each module should be on a separate line as it was before